[Talent Pool] JobSeekerProfileCard import fails in BrowseCompany.tsx on case sensitive FS

**Describe the bug**
`JobSeekerProfileCard` import fails in `BrowseCompany.tsx`:

https://github.com/talent-connect/connect/blob/dfae5c436e8379b9829a7565bacbd9a7abdc5f2b/apps/redi-talent-pool/src/pages/app/browse/BrowseCompany.tsx#L22

on case sensitive file system (Windows WSL2/Ubuntu20.04), with error:

```
ERROR in ./pages/app/browse/BrowseCompany.tsx
Module not found: Error: Can't resolve '../../../components/organisms/JobSeekerProfileCard' in '/home/user/github/connect/apps/redi-talent-pool/src/pages/app/browse'
ERROR in /home/user/github/connect/apps/redi-talent-pool/src/pages/app/browse/BrowseCompany.tsx(22,38):
TS2307: Cannot find module '../../../components/organisms/JobSeekerProfileCard' or its corresponding type declarations.
Version: typescript 4.1.6
```

The file is named `JobseekerProfileCard.tsx` (lower case `s`).

**To Reproduce**
Steps to reproduce the behavior:
1. Have your repo in a Windows WSL2/Ubuntu20.04 file system (not in `/mnt/c/`)
1. Run `yarn start:alone:talent-pool`
1. See error

**Expected behavior**
Running `yarn start:alone:talent-pool` would complete without errors

**Desktop:**
 - OS: Windows 10 / WSL2 Ubuntu 20.04

--- a/apps/redi-talent-pool/src/pages/app/browse/BrowseCompany.tsx
+++ b/apps/redi-talent-pool/src/pages/app/browse/BrowseCompany.tsx
@@ -19,7 +19,7 @@ import {
   useQueryParams,
   withDefault,
 } from 'use-query-params'
-import { JobseekerProfileCard } from '../../../components/organisms/JobSeekerProfileCard'
+import { JobseekerProfileCard } from '../../../components/organisms/JobseekerProfileCard'
 import { LoggedIn } from '../../../components/templates'
 import { useBrowseTpJobseekerProfilesQuery } from '../../../react-query/use-tpjobseekerprofile-query'
 

